checks/postgres/check.go

```diff
@@ -41,7 +42,12 @@ func New(config Config) func() error {
 			return err
 		}
 
-		_, err = db.Query(`SELECT VERSION()`)
+		rows, err := db.Query(`SELECT VERSION()`)
+		defer func() {
+			if err = rows.Close(); err != nil {
```

Comment:
`rows` can be `nil` at this point when for example `err` is not

Reply:
fixed